### PR TITLE
add "Accept" header to bing.py

### LINKED ISSUE #3466: No Bing results for Searx (but for SearxNG)
The public instances Searx instances don't seem return any results for Bing but SearxNG instances do. I think the situation has been the same for days or even weeks.

I tried out several searx and searxng instances: https://searx.space/

### Searx

Example Searx instance: https://searx.roflcopter.fr/ (1.1.0-53-c647b55e)
[Bing search in roflcopter.fr](https://searx.roflcopter.fr/search?q=%21bi%20foo&categories=none&language=en-US) gave "Sorry! we didn't find any results."

### SearxNG

Example SearxNG instances: https://northboot.xyz/ (2023.2.4+7320b0c7)
[Bing search in northboot.xyz](https://northboot.xyz/search?q=%21bi+foo&category_general=1&language=en-US&time_range=&safesearch=1&theme=simple) gave a pageful of results.

### Suggestion

Might the fix be included in the [SearxNG commit list](https://github.com/searxng/searxng/commits/master/searx/engines/bing.py)? There are several references to Bing that are not included in [Searx commit list](https://github.com/searx/searx/commits/master/searx/engines/bing.py)
There is a big diff between https://raw.githubusercontent.com/searxng/searxng/master/searx/engines/bing.py and https://raw.githubusercontent.com/searx/searx/master/searx/engines/bing.py

### PR DESCRIPTION
**Pull request by glanham-jr**
## What does this PR do?

Add the Accept header in `bing.py` with relevant information. 

## Why is this change important?

Bing stopped returning results unless the Accept header was set to the specified line.

## How to test this PR locally?

Run the development instance and search `!bi foo`. There should now be results showing up.

## Related issues

Closes #3466

### CODE REVIEW

File: searx/engines/bing.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ def request(query, params):
     params['url'] = base_url + search_path
     params['headers']['User-Agent'] = ('Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 '
                                        '(KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36')
+    params['headers']['Accept'] = 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8'
```
Comment (juusoohtonen) [inline searx/engines/bing.py:55]: The same `Accept` header is already present in searxng: https://github.com/searxng/searxng/blob/7fc8d72889742399abb0d327e355ab350ddb395b/searx/engines/bing.py#L73
So, I'm quite confident this change does not break anything.
However, I did not test the fix myself locally.

**Review by juusoohtonen — APPROVED** [review]

**Review by juusoohtonen — DISMISSED** [review]

**Comment by juusoohtonen** [comment]
Could we have this PR merged and the fix released?

Whether the change fixes also other search engines is another topic, [discussed separately](https://github.com/searx/searx/discussions/3476).

**Comment by br4nnigan** [comment]
this only works if the user agent header is removed

**Comment by glanham-jr** [comment]
@br4nnigan Removed the User Agent header

**Comment by juusoohtonen** [comment]
Do you have any idea why removing User Agent was required? I woudn't mind seeing explanation (why) in the commit messages. Anyway, the implementation on the header part is now similar to SearxNG (https://github.com/searxng/searxng/blob/7fc8d72889742399abb0d327e355ab350ddb395b/searx/engines/bing.py#L72,L73), so this PR is good to go.

**Review by juusoohtonen — APPROVED** [review]

**Review by juusoohtonen — APPROVED** [review]

**Review by kvch — APPROVED** [review]

**Pull request merged**